main.go

```diff
@@ -225,6 +235,16 @@ func main() {
 	}
 }
 
+func initializeProtocol(stdin io.Reader, stdout io.Writer) {
+	workerRunnerTranport := protocol.NewStdioTransport()
```

Comment:
```suggestion
	workerRunnerTransport = protocol.NewStdioTransport()
```

Reply:
!!! that is a bug I wish go would warn me about!